### PR TITLE
[#174418447] Dropdown to create resolution/dialog

### PR DESCRIPTION
**Pull request by DmitryBarskov**
https://www.pivotaltracker.com/story/show/174418447

https://projects.invisionapp.com/d/main?origin=v7#/console/20307299/427358479/preview?scrollOffset=1244

### CODE REVIEW

File: plugin.rb [hunk 1]
```diff
@@ -18,6 +18,7 @@
   "stylesheets/common/resolution-form.scss",
   "stylesheets/common/landing.scss",
   "stylesheets/common/communities-page.scss",
+  "stylesheets/common/new-dropdown.scss",
```
Comment (Vadim-Kurnatovskiy) [inline plugin.rb]: may be change file name to `new-topic-dropdown`? the name is not clear.

**Comment by DmitryBarskov** [comment]
@IlgizGilmanov could you help me with the styles?

**Comment by IlgizGilmanov** [comment]
> @IlgizGilmanov could you help me with the styles?

Sure

**Review by ShaidullinArtur — APPROVED** [review]
looks :ok except css styles, but I think that we can fix them in another PR

**Comment by IlgizGilmanov** [comment]
With dropdown styles everything is ok. The only thing is, can we move this button to the right, above the dialog box (we touched on this issue at one of the stand-ups), and also move and show the edit button (wrench) to the left of it?

**Review by ShaidullinArtur — APPROVED** [review]

**Review by Vadim-Kurnatovskiy — APPROVED** [review]

**Review by IlgizGilmanov — APPROVED** [review]

**Pull request merged**